### DISCUSSION TITLE
How to env "unuse" pkg?

### DISCUSSION THREAD

**Discussion by MaxforCherubim** [x-cmd/x-cmd#69] [Q&A] [answered]
I had `x env use jieba`, and the jieba already loaded in my path.
I forgot to use the command `x env try jieba`, so how to make the jieba be a 'try'?
Or env "unuse" pkg.

**Comment by qiakai** [comment, marked as answer]
```sh
x env unuse jieba
x env try jieba
```

**Reply by MaxforCherubim** [reply]
wo, I had seen it!
[https://cn.x-cmd.com/mod/env/unuse#x-env-unuse](url)
That is embarrassed.😳
